spec-main/api-net-spec.ts

```diff
@@ -1393,6 +1393,29 @@ describe('net module', () => {
       await collectStreamBody(nodeResponse);
       expect(nodeRequestProcessed).to.equal(true);
     });
+
+    it('should correctly throttle an incoming stream', async () => {
+      let numChunksSent = 0;
+      const serverUrl = await respondOnce.toSingleURL((request, response) => {
+        const data = randomString(kOneMegaByte);
+        const encoding = 'utf8';
+        const write = () => {
+          let ok = true;
+          do {
+            numChunksSent++;
+            if (numChunksSent > 3000) return;
```

Comment:
I think we can turn this down a bit.

```suggestion
            if (numChunksSent > 30) return;
```